### PR TITLE
Update product taskmaster and linting

### PR DESCRIPTION
**Pull request by paulmskim**
### 🎫 Ticket

N/A

### 🗒️ Description

* Updates product taskmaster version
* Adds `.eslintrc` and `.stylelintrc` files that extend from product taskmaster
* Uses product taskmaster gulp tasks to lint JS and PostCSS
* Adds github action to run linting on PR
* Fixes the JS and PostCSS lint issues

### 🎥 Artifacts 
image

### ✔️ Checklist
- [ ] I've included a changelog entry both in readme.txt and changelog.txt files. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

### CODE REVIEW

**Comment by paulmskim** [comment]
On hold until we get variables for `rgba()` fixed

**Comment by paulmskim** [comment]
@bordoni @juanfra this is ready for PR

File: src/resources/js/v2/tickets-block.js [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since 5.0.3
+ *
+ * @type {PlainObject}
+ */
 tribe.tickets = tribe.tickets || {};
 
+/**
+ * Configures RSVP Tooltip Object in the Global Tribe variable
```
Comment (juanfra) [inline src/resources/js/v2/tickets-block.js:11]: This one seems wrong.

**Review by juanfra — APPROVED** [review]
Looking good!

File: src/resources/js/v2/tickets-block.js (hunk 1 above)
Comment (paulmskim) [reply src/resources/js/v2/tickets-block.js:11]: good catch, will fix this.

**Review by juanfra — APPROVED** [review]
⚡

**Pull request merged**